go/toolchains.rst

```diff
@@ -296,7 +296,12 @@ This downloads a Go SDK for use in toolchains.
         name = "go_sdk",
         goos = "linux",
         goarch = "amd64",
-        version = "1.12.5",
+        version = "1.18.1",
+        sdks = {
+            # You can get the expected filenames and checksums from https://go.dev/dl/
```

Comment:
Can you make it a bit more clear that these are not necessary all the time and further elaborate on what the circumstances are when you'd want to override these

Reply:
Updated the comment. PTAL.